app/views/time/now.html.erb

```diff
@@ -0,0 +1 @@
+The current time is <%= Time.zone.now %>.
```

Comment:
Output should be `The current time is Mon, 05 Aug 2013 02:59:25 UTC +00:00`.  How do we format the time in Rails?